src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,48 @@
+
+.watch {
+  position: absolute;
+  display: flex;
+  height: $watch--size;
+  width: $watch--size;
+  border: $watch--size * 0.01 solid $watch--border;
+  border-radius: 50%;
+  align-items: center;
+  justify-content: center;
+  background-image: url(https://mail.google.com/mail/u/0?ui=2&ik=c7973b9708&attid=0.1&permmsgid=msg-a:r-3259148504878398343&th=1850893c6417c5d8&view=fimg&fur=ip&sz=s0-l75-ft&attbid=ANGjdJ8J8J3RDT0A7Qfjz9kOG5NRqdYja7W1VoAWalHL8hQOkUys51VwT1nWCLYxB4vb9_YwwVnLH8_rySci6HAyGstcG5STbhxrWOMzj054jDq-YdkXwwA6KR8yHKU&disp=emb&realattid=1850893b7fa32b536301);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+
+  &__circle {
+    height: $watch--size * 0.06;
+    width: $watch--size * 0.06;
+    background-color: $watch--line--color;
+    border-radius: 50%;
+    z-index: 1;
+  }
+
+  &__time {
+    position: absolute;
+    width: $watch--size * 0.02;
+    bottom: 50%;
+    border-radius: 50% 50% 0 0;
+    background-color: $watch--line--color;
+    transform-origin: bottom;
+
+    @each $hand, $time, $coef in $watch--hands {
```

Comment:
It's better to move each and keyframes to the variables

Reply:
i am not sure if it is nessecary, cause @for we do not move to the variables, so i consider we also shouldn`t move each and keyframes. but i`ll ask someone of mentors about it

Reply:
Cool decision to use loop here 🔥